### DISCUSSION TITLE
Help needed to migrate gestion data between two NC servers.

### DISCUSSION THREAD

**Discussion by TheMiNuS** [baimard/gestion#395] [Features request]
Hello,

I would like to move all my data from a NC instance to another one.
Am I missing something ? I do not find a way to reimport the data (ok for the backup).

Thx
MiNuS

**Comment by TheMiNuS** [comment]
I look at the roadmap thread and I don't see anything about backups import.
@baimard do you plan to release that feature or it's something too complex to implement for now.

I started using gestion on the wrong NC instance and I would likt to move away from it. If there is a way to do it at SQL level without corrupting anything let me know as I can also do that without any problem.

**Reply by baimard** [reply]
Dear,

I can implement it on the next release if you want.

If you want to try by your hands. Just copy all gestion_ table in new instance and everything gone be good !

**Comment by ghost** [comment]
I admit that it would be useful to have an export to file / import from file  tool, so we can avoid messing with the database. This is ok for testing and dev, but from a normal user perspective, this is not really viable...

thanks for the last release.

**Comment by baimard** [comment]
Gone change the roadmap and work on it now !

**Reply by TheMiNuS** [reply]
I see on the roadmap that your plan was to release Export/Import on 2.6.3.
Is it still the case as I don't see it in changelog and after upgrade I still have the import feature disabled.

Something intresting that could be implemented is a scheduled backup/export as cospend is offering. It's a very cool feaure to secure your exports without having to do it manually.

Thanks
